analysers/analyser_merge_power_substation_minor_FR.py

```diff
@@ -57,9 +57,40 @@ def __init__(self, config, logger = None):
                     static2 = {
                         "substation": "minor_distribution", # Currently default value, we're unable to destinguish distribution and minor_distribution in opendata
                         "voltage": "20000"}, # Currently lawful default value as there is no opendata to define it. Mappers may be knowledgeable
+                    mapping1 = {
+                        "operator": lambda res: self.extract_operator.get(res['GRD'])[0] if res['GRD'] in self.extract_operator else res['GRD'],
+                        "operator:wikidata": lambda res: self.extract_operator.get(res['GRD'])[1] if res['GRD'] in self.extract_operator else None,
+                        "name": lambda fields: fields["Nom poste"] if fields["Nom poste"] != "" else None},
                     mapping2 = {
-                        "operator": "GRD",
-                        "name": lambda fields: fields["Nom poste"] if fields["Nom poste"] != "" and fields["GRD"] != "Strasbourg Électricité Réseaux" else None,
-                        "ref": lambda fields: fields["Nom poste"] if fields["Nom poste"] != "" and fields["GRD"] == "Strasbourg Électricité Réseaux" else None,
                         "source": lambda fields: self.source() + " - " + fields["GRD"]},
                 )))
+
+    # Main source: https://wiki.openstreetmap.org/wiki/Power_networks/France/Exploitants#Entreprises_de_distribution
+    extract_operator = {
```

Comment:
Hello @Jean-BaptisteC it is an intersecting list between values in the ORE dataset and the operators list in the wiki.
Commented values aren't known from us

Reply:
I've added all your suggestions @Jean-BaptisteC